shared/elements/gaia_switch/style.css

```diff
@@ -15,7 +15,7 @@ gaia-switch {
   font-size: 1.9rem;
   color: var(--label-color);
   padding: 1rem 0;
-  line-height: 3rem;
+  line-height: 2.6rem;
```

Comment:
What is the reason for this change? This would cause it to diverge from the other components, https://github.com/mikehenrty/gaia/blob/bug-1205040-tracking-protection-setting/shared/elements/gaia_radio/style.css#L25

(We should probably implement a gaia-label component in the future)